fastparquet/api.py

```diff
@@ -122,36 +122,35 @@ def row_group_filename(self, rg):
         else:
             return self.fn
 
-
     def read_row_group_file(self, rg, columns, categories, index=None,
-                            assign=None):
+                            assign=None, mr_times=[]):
```

Comment:
The term `mr_times` seems odd to me.  Perhaps something about datetime encoding scheme?  I suspect that this term is used for historical reasons, but a new user may not know that `mr_times` means "possibly 96-bit datetime encodings".

Also, are more keyword arguments expected?  Would it makes sense to use `**kwargs` here instead of passing down new keywords every time something like this occurs?  Generally I'm looking for a way to keep small additions like this cheap to maintain in the future.